lib/internal/repl.js

```diff
@@ -124,6 +125,7 @@ function setupHistory(repl, historyPath, oldHistoryPath, ready) {
   }
 
   function onread(err, data) {
+    var threw;
```

Comment:
Nit: why not putting this in the `if` branch where the `try...catch` is defined?